src/views/RegisterPage/index.js

```diff
@@ -17,19 +25,39 @@ class RegisterPage extends Component {
       confirmpassword: undefined,
       termsandconditions: undefined
     };
+    this.handleSubmit = this.handleSubmit.bind(this);
```

Comment:
you can actually avoid calling .bind(this) with every method by using arrow functions! more [here](https://medium.com/@nikolalsvk/loosing-bind-this-in-react-8637ebf372cf) but essentially `handleSubmit(event) { ..` becomes `handleSubmit = (event) => { ... ` and everything else stays the same